.github/workflows/node-ci.yml

```diff
@@ -19,4 +19,5 @@ jobs:
           node-version: 16
           cache: npm
       - run: npm ci
+      - run: npx prettier-standard --check
```

Comment:
i'd like the checks that run in ci to align as much as possible with those run locally before pushing. could this be included in the list run by `npm test` (possibly as the `lint:js` script)?

Reply:
I notice that the `npm test` command appears to run all the `lint:*` commands, so presumably we'll both check *and* reformat... is this OK?

Reply:
I tested this, and in fact it runs the `lint:js` task first, so if there are formatting problems it fails and stops there, and never runs the `list:js:fix` task.

So I think we get the behaviour you're looking for.

We maybe need to document somewhere that people can run `npm run lint:js:fix` to sort out formatting issues. Where would be a good place for that?